packages/esm-ward-app/src/ward-workspace/patient-transfer-bed-swap/patient-transfer-swap.scss

```diff
@@ -0,0 +1,70 @@
+@use '@carbon/styles/scss/spacing';
+@use '@carbon/styles/scss/type';
+@use '@carbon/layout';
+@import '~@openmrs/esm-styleguide/src/vars';
+
+.workspaceContent {
+  padding: spacing.$spacing-05;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  color: #393939;
+}
+
+.field {
+  margin-bottom: layout.$spacing-05;
+  & > h2 {
+    margin-bottom: layout.$spacing-03;
+  }
+}
+
+.contentSwitcher {
+  margin-top: layout.$spacing-03;
+}
+
+.workspaceForm {
+  margin-top: spacing.$spacing-05;
```

Comment:
Let's stick to using `layout`. Means we can remove the `@use '@carbon/styles/scss/spacing` import.

Reply:
This was an older change, I'll change this, thanks!